Sources/Kommunicate/Classes/Utilities/KMCustomisation.swift

```diff
@@ -0,0 +1,109 @@
+                    } else if(key.elementsEqual(":file")) {
+                        attachments.append(AttachmentType.document)
+                    } else if(key.elementsEqual(":audio")) {
+                        Kommunicate.defaultConfiguration.hideAudioOptionInChatBar = value
+                    }
+                }
+                Kommunicate.defaultConfiguration.chatBar.optionsToShow = .some(attachments)
+            }
+            if let maxAttachmentAllowed = settingDict["maxAttachmentAllowed"] as? Int {
+                Kommunicate.defaultConfiguration.chatBar.photosSelectionLimit = maxAttachmentAllowed
+            }
+            if let showTopbarStartConversationButton = settingDict["showTopbarStartConversationButton"] as? Bool {
+                Kommunicate.defaultConfiguration.hideStartChatButton = !showTopbarStartConversationButton
+            }
+            if let showStartNewConversation = settingDict["showStartNewConversation"] as? Bool {
+                Kommunicate.defaultConfiguration.hideBottomStartNewConversationButton = !showStartNewConversation
+            }
+            if let refreshOption = settingDict["refreshOption"] as? Bool {
+                Kommunicate.defaultConfiguration.isRefreshButtonEnabled = refreshOption
+            }
+            if let restrictMessageTypingWithBots = settingDict["restrictMessageTypingWithBots"] as? Bool {
+                Kommunicate.kmConversationViewConfiguration.restrictMessageTypingWithBots = restrictMessageTypingWithBots
+            }
+            
+            if let oneTimeRating = settingDict["oneTimeRating"] as? Bool {
+                Kommunicate.defaultConfiguration.oneTimeRating = oneTimeRating
+            }
+            if let hideSenderName = settingDict["hideSenderName"] as? Bool {
+                KMCellConfiguration.hideSenderName = hideSenderName
+            }
+            if let hideBackButtonInConversationList = settingDict["hideBackButtonInConversationList"] as? Bool {
+                Kommunicate.defaultConfiguration.hideBackButtonInConversationList = hideBackButtonInConversationList
+            }
+            if let disableRichMessageButtonAction = settingDict["disableRichMessageButtonAction"] as? Bool {
+                Kommunicate.defaultConfiguration.disableRichMessageButtonAction = disableRichMessageButtonAction
+            }
+            if let isNewSystemPhotosUIEnabled = settingDict["isNewSystemPhotosUIEnabled"] as? Bool {
+                Kommunicate.defaultConfiguration.isNewSystemPhotosUIEnabled = isNewSystemPhotosUIEnabled
+            }
+            if let restrictedWordRegex = settingDict["restrictedWordRegex"] as? String, !restrictedWordRegex.isEmpty {
+                Kommunicate.defaultConfiguration.restrictedMessageRegexPattern = restrictedWordRegex
+            }
+            
+        } catch let error as NSError {
+            return error.description
+        }
+        
+        return "Successfully created settings"
```

Comment:
we do we need to return string, either you can create enum for the status with that you can pass the error string or you can use boolean along with error statement